genrl/deep/agents/ddpg/ddpg.py

```diff
@@ -128,16 +128,17 @@ def __init__(
         self.create_model()
 
     def create_model(self):
-        state_dim = self.env.observation_space.shape[0]
-        action_dim = self.env.action_space.shape[0]
+        # state_dim = self.env.observation_space.shape[0]
+        # action_dim = self.env.action_space.shape[0]
+        state_dim, action_dim, self.discrete = self.get_env_properties()
```

Comment:
Not related to build errors

Discrete should always be False here (DDPG, TD3, SAC). So maybe we should add a condition to check if self.discrete is True. If not, raise error? What do you think?

Reply:
yea, correct this is why CartPole was working :(